`cargo test` compilation error

In my multi-module project while I am running `cargo test` in the root directory, I'm getting:

```
>cargo test
   Compiling winapi v0.3.9
   Compiling wepoll-sys v3.0.0
   Compiling libz-sys v1.1.2
   Compiling cmake v0.1.44
   Compiling rdkafka-sys v2.1.0+1.5.0
   Compiling testcontainers v0.11.0
error[E0433]: failed to resolve: could not find `fmt` in `core`
  --> C:\Users\user\.cargo\registry\src\github.com-1ecc6299db9ec823\testcontainers-0.11.0\src\core\docker.rs:74:10
   |
74 | #[derive(derivative::Derivative)]
   |          ^^^^^^^^^^^^^^^^^^^^^^ could not find `fmt` in `core`
   |
   = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to previous error

For more information about this error, try `rustc --explain E0433`.
error: could not compile `testcontainers`.

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: build failed
```

OS: Windows 10

There is no error while running tests in submodules. Also, there is no error if I use `0.9.1`.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -10,7 +10,6 @@ repository = "https://github.com/testcontainers/testcontainers-rs"
 version = "0.11.0"
 
 [dependencies]
-derivative = "2"
 hex = "0.4"
 hmac = "0.9"
 log = "0.4"

--- a/src/core/docker.rs
+++ b/src/core/docker.rs
@@ -1,5 +1,5 @@
 use crate::{Container, Image};
-use std::{collections::HashMap, io::Read};
+use std::{collections::HashMap, fmt, io::Read};
 
 /// Defines the minimum API required for interacting with the Docker daemon.
 pub trait Docker
@@ -71,11 +71,13 @@ impl Ports {
 }
 
 /// Log streams of running container (stdout & stderr).
-#[derive(derivative::Derivative)]
-#[derivative(Debug)]
 pub struct Logs {
-    #[derivative(Debug = "ignore")]
     pub stdout: Box<dyn Read>,
-    #[derivative(Debug = "ignore")]
     pub stderr: Box<dyn Read>,
+}
+
+impl fmt::Debug for Logs {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("Logs").finish()
+    }
 }

